ganga/GangaCore/__init__.py

```diff
@@ -961,3 +961,32 @@ def filefilter(fn):
 cred_config = makeConfig('Credentials', 'This configures the credentials singleton')
 cred_config.addOption('CleanDelay', 1, 'Seconds between auto-clean of credentials when proxy externally destroyed')
 cred_config.addOption('AtomicDelay', 1, 'Seconds between checking credential on disk')
+
+# ------------------------------------------------
+# Database
+db_config = makeConfig("DatabaseConfigurations",
+                       "Selection of database for ganga")
+db_config.addOption("database", "MONGODB",
+                    "others have not been implemented yet")
+db_config.addOption(
+    "containerName", "ganga_mongomon", "the identifier used to tag the docker container"
+)
+db_config.addOption("username", "default", "username")
+db_config.addOption("password", "default", "password")
```

Comment:
What is the role of this username and password? Is it such that another user on the same machine will not be able to communicate with the server? We do not want users to edit these themselves, so what is the best approach here?

Reply:
>  Is it such that another user on the same machine will not be able to communicate with the server?

Yes.

> We do not want users to edit these themselves
Yes, we would not want that. That is why we will create a password different from the default Mongo password. In any case, the user explicitly wants to access the data, it can be done after reading the appropriate password from the config.

Reply:
OK, I see. So you will create a random password. So once created this password will stay the same the next time the server is started? I assume that it is only for the server connection and that the information itself in the database is not encrypted (so if password is lost, there is no loss of information).